spec/presenters/step_by_step_presenter_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "rails_helper"
+require "gds_api/test_helpers/link_checker_api"
+
+RSpec.describe StepByStepPresenter do
+  include GdsApi::TestHelpers::LinkCheckerApi
+  include AuthenticationControllerHelpers
```

Comment:
I don't think you need to explicitly include `AuthenticationControllerHelpers` here as it's already being included here: https://github.com/alphagov/collections-publisher/blob/master/spec/support/authentication.rb#L30

Reply:
We're testing a presenter rather than a controller, so it isn't getting pulled in automatically. Without this line I get errors along the lines of `unknown method stub_user`.